docs/pages/_app.tsx

```diff
@@ -4,9 +4,13 @@ import * as React from 'react';
 import BreakPointDebug from '../components/BreakPointDebug/BreakPointDebug';
 import { Footer } from '../components/Footer/Footer';
 import Header from '../components/Header/Header';
+import { requireDecoderDuringBuild } from '../RequireDecoderDuringBuild';
 import '../styles/globals.css';
+import { navTreeDecoder } from '../Types/NavTree/Decoder';
 
 const App: React.FC<AppProps> = ({ Component, pageProps }) => {
+  // TODO: render the navTree
+  const navTree = requireDecoderDuringBuild(navTreeDecoder)(pageProps.navTree);
```

Comment:
@SamEdwards-1: here you go, the `_app` component is guaranteed (at build time) to have a `navTree` object you can use to render the navigation.

Reply:
@spejamchr Looks good to me, lets merge it in